services/common/src/wrapper/v0/mod.rs

```diff
@@ -0,0 +1,806 @@
+//! Implementation of the pool service backend which interacts with
```

Comment:
This isn't specific for the pool service.

Reply:
Done